templates/taxi/car_detail.html

```diff
@@ -12,7 +12,19 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>Drivers
+    {% if user in car.drivers.all %}
+      <form action="{% url 'taxi:delete-me-car' pk=car.id %}" method="post" class="link-to-page">
+        {% csrf_token %}
+        <input class="btn btn-danger" type="submit" value="Delete me from this car">
+      </form>
+    {% else %}
+      <form action="{% url 'taxi:assign-me-car' pk=car.id %}" method="post" class="link-to-page">
+        {% csrf_token %}
+        <input class="btn btn-success" type="submit" value="Assign me to this car btn">
```

Comment:
The button label 'Assign me to this car btn' seems to have an extra word 'btn'. It should be 'Assign me to this car' for clarity and consistency.